sample_apps/simple_app_python/example_app.py

```diff
@@ -178,8 +173,9 @@ def do_main(args) -> bool:
             print(fnl(), 'warm_restart() failed\n')
             sys.exit(1)
 
-        result = server_dispatch(PROVISIONING_DIR,
-                                 server_app_host, server_app_port)
+        write_certficates_to_file(cctm, 'server', PROVISIONING_DIR, print_all)
```

Comment:
Only certificate data will be written out to known-location in provisioning dir by this method.

Reply:
There is a typo in the function name: `write_certficates_to_file -> write_certificates_to_file`